build.gradle

```diff
@@ -21,7 +21,7 @@ buildscript {
             opensearch_build += "-SNAPSHOT"
         }
         common_utils_version = System.getProperty("common_utils.version", opensearch_build)
-        kotlin_version = System.getProperty("kotlin.version", "1.6.0")
```

Comment:
are these coming from upstream somewhere/so we don't have to manually change it every time?